Examples/BookingSystem.AspNetFramework/Stores/FacilityStore.cs

```diff
@@ -78,29 +78,29 @@ public FacilityStore(bool UseSingleSellerMode)
                                 validFromStartDate: isValid);
                             break;
                         case TestOpportunityCriteriaEnumeration.TestOpportunityBookableNonFreePrepaymentOptional:
-                            (facilityId, slotId) = FakeBookingSystem.Database.AddClass(
```

Comment:
Changes from 81 to 103 are not things I did in this PR, but made when Core -> Framework script was run